package.json

```
@@ -78,6 +79,9 @@
     "prettier": "prettier \"**/*.{md,css,scss,yaml,yml,html}\"",
     "pretest": "yarn lint",
     "test": "mocha",
+    "pre:cy:genfix": "node ./generate-fixture.js",
+    "cy:dev:test": "cypress run",
+    "cy:dev:watch": "cypress open",
```

```suggestion
    "cy:dev:test": "node ./generate-fixture.js && cypress run",
    "cy:dev:watch": "node ./generate-fixture.js && cypress open",
```
For the pre script to run before the cypress scripts, it needs to match their names.  Since there are only two Cypress scripts, I think it's fine to have a bit of repetition here.